torch/linalg/__init__.py

```diff
@@ -419,12 +452,12 @@
              in the real case or by :math:`e^{i \phi}` for any :math:`\phi \in \mathbb{R}` in the
              complex case, and the resulting singular vectors will give a different
              eigendecomposition of the matrix.
-             As such, the eigenvectors are not continuous functions of the inputs, and different
-             platforms, like NumPy, or inputs on different devices, may produce different
-             eigenvectors.
+             As this choice is not canonical, it happens that the eigenvectors are not continuous
```

Comment:
See previous proposal for simplifying sections like this to avoid some tricky language.